CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## master
 
+- Add `fail_when_no_matches` parameter to `match` method. ([@Earendil95][])
```

Comment:
You should also add a value to this link in the end of the document.

Reply:
Oh, thank you for reminding me :)